CODEOWNERS

```diff
@@ -90,6 +90,8 @@
 #   Review all Cilium and Hubble code related to observing system events,
 #   exporting those via gRPC protocols outside the node and outside the
 #   cluster. those event channels, for example via TLS.
+# - @cilium/hubble-metrics:
+#   Review code related to Hubble metrics.
```

Comment:
Could we be more specific here? What I would suggest is to remove `sig-hubble-api` from the codeowners for Hubble metrics and add responsibility to the hubble-metrics group to ensure changes in exposed metrics are consistent and not breaking without careful consideration.